src/com/rollbar/notifier/RollbarNotifier.as

```diff
@@ -131,8 +131,8 @@ package com.rollbar.notifier {
             return handleError(newError);
         }
 
-        public function handleOtherEvent(event:*):String {
-            var newError:Error = new Error("A non-Error or ErrorEvent was thrown and not caught: " + event.toString());
+        public function handleOtherEvent(error:*):String {
+            var newError:Error = new Error("A non-Error or ErrorEvent was thrown and not caught: " + error.toString());
```

Comment:
Event will always be "[UncaughtErrorEvent type="uncaughtError" bubbles=true cancelable=true eventPhase=2]" so I don't see why sending it really. :)